test/e2e/browsers.conf.js

```diff
@@ -2,7 +2,14 @@ const browsers = require('../browsers.conf')
 
 module.exports = [
   browsers['SAFARI'],
-  browsers['EDGE'],
+  {
+    ...browsers['EDGE'],
+    // Without this specific version, execute and executeAsync fails on Edge.
+    'browserstack.selenium_version': '3.5.2',
+    // Note: more recent versions of selenium 3 have the same issue, but selenium 4 seems to be
+    // fine:
+    // 'browserstack.selenium_version': '4.0.0-alpha-6',
```

Comment:
did you find any discussion on that?

Reply:
I found [this](https://github.com/webdriverio/webdriverio/issues/5513) but it doesn't seem related to remote selenium usages, just `standalone-selenium`. I found the working selenium version by using https://www.browserstack.com/automate/capabilities (this is the default selenum version output in the capabilities).